services/viva-ms/lambdas/createVivaCase.js

```diff
@@ -59,7 +58,7 @@ export async function main(event) {
     return console.error('(Viva-ms) Viva Application WorkflowId not present in response, aborting');
   }
 
-  const [getUserCaseFilteredOnWorkflowIdError, caseItem] = await to(
+  const [getUserCaseFilteredOnWorkflowIdError, caseItemExists] = await to(
```

Comment:
Naming it `Exists` implies that it is a bool. `caseItem` is fine.

Reply:
You're a true champ. Tx! Fix this now